packages/ui/components/DataFeedDetails.vue

```diff
@@ -90,6 +85,15 @@ export default {
     feedAddress() {
       return this.feed.address
     },
+    feedAddressUrl() {
+      if (this.feed.network === 'conflux') {
+        return `https://testnet.confluxscan.io/address/${this.feedAddress}`
+      } else if (this.feed.network === 'mainnet') {
+        return `https://etherscan.io/address/${this.feedAddress}#code`
+      } else {
+        return `https://${this.feed.network}.etherscan.io/address/${this.feedAddress}#code`
+      }
+    },
```

Comment:
When we add support for Boba, please let's make all the business logic network-independent. Let's include URL templates and the network colors into each feed entry in the JSON config. I don't want us to do a PR and rebuild the docker image every time we want to add a new network.

Reply:
Or if you prefer this suggestion not to have any impact on this sprint's scope, we can integrate Boba in this same way, and then next sprint refactor the thing so the next ones (e.g. Celo, Polygon, BNB, Aurora, Harmony, etc.) are easier.

Reply:
I have opened #55 for this

Reply:
Thanks!